### PR TITLE
:sparkles: Introduce Alexa Conversations

### PR DESCRIPTION
**Pull request by rubenaeg**
## Proposed changes
This PR introduces Alexa Conversations, along with improvements to the deploy process of an Alexa skill.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Review by aswetlow — APPROVED** [review]

File: platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+export interface ApiResponseOutputOptions extends OutputOptions {
+  apiResponse: Record<string, string | number | boolean>;
+}
+
+@Output()
+export class ApiResponseOutput extends BaseOutput<ApiResponseOutputOptions> {
+  build(): OutputTemplate | OutputTemplate[] {
+    return {
+      listen: this.options.listen,
```
Comment (jankoenig) [inline platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts:11]: `listen` is a [reserved property](https://www.jovo.tech/docs/output-classes#reserved-properties), so it doesn't necessarily need to be added to this output. By default, it's `true`

File: platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      platforms: {
+        alexa: {
+          nativeResponse: {
+            version: '1.0',
```
Comment (jankoenig) [inline platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts:15]: `version` and `sessionAttributes` isn't needed here, will be added automatically

File: platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts (hunk 1 above)
Comment (jankoenig) [reply platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts:11]: never mind, I think this is OK and clearer if we keep it in there

File: platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts (hunk 2 above)
Comment (jankoenig) [reply platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts:15]: let's keep it in there for now

**Review by jankoenig — APPROVED** [review]

**Pull request merged**